src/test/java/org/broadinstitute/hellbender/tools/picard/analysis/CollectBaseDistributionByCycleIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.tools.picard.analysis;
+
+    import htsjdk.samtools.metrics.MetricsFile;
+    import org.broadinstitute.hellbender.CommandLineProgramTest;
+    import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+    import org.testng.Assert;
+    import org.testng.annotations.Test;
+
+    import java.io.File;
+    import java.io.FileReader;
+    import java.io.IOException;
+
+public final class CollectBaseDistributionByCycleIntegrationTest extends CommandLineProgramTest {
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "picard/analysis/CollectBaseDistributionByCycle");
+
+    @Test
+    public void test() throws IOException {
+        final File input = new File(TEST_DATA_DIR, "first5000a.bam");
+        final File expectedFile = new File(TEST_DATA_DIR, "CollectBaseDistributionByCycle.txt");
+        final File outfile = File.createTempFile("testCollectBaseDistributionByCycle", ".metrics");
+        final File pdf = File.createTempFile("testCollectBaseDistributionByCycle", ".pdf");
+        outfile.deleteOnExit();
+        pdf.deleteOnExit();
+        final String[] args = new String[]{
+                "--INPUT", input.getAbsolutePath(),
+                "--OUTPUT", outfile.getAbsolutePath(),
+                "--CHART", pdf.getAbsolutePath(),
```

Comment:
Do we want to be testing the plot production?  I thought our policy was going to be that plots are untested and unsupported?